easybuild/tools/docs.py

```diff
@@ -744,28 +744,65 @@ def list_toolchains_rst(tcs):
     """ Returns overview of all toolchains in rst format """
     title = "List of known toolchains"
 
-    # figure out column names
-    table_titles = ['name', 'compiler', 'MPI']
-    for tc in tcs.values():
-        table_titles.extend(tc.keys())
+    # Specify the column names for the table
+    table_titles = ['name', 'COMPILER', 'MPI', 'linalg', 'FFT']
```

Comment:
Should be `'compiler'` to ensure correct column name is used in the end, no?

Now you get `COMPILER` as column name, I think...

Reply:
Ah, you are right.  I think table_titles is OK, but the change should be to
```
    col_names = {
        'name': 'Name',
        'COMPILER': 'Compiler(s)',
        'linalg': "Linear algebra",
    }
```
to match the change you requested to `table_titles` before.  Sorry.  Will do that.

Reply:
I think it looks awkward to have `name` vs `COMPILER` in `table_titles`.

For `MPI` & `FFT` it's OK, since those are abbreviations.

Reply:
They are just placeholders internal to the code.  How about _all_ of the entries in `table_titles` go upper, and the `col_names` are used to set the headings in the display?  I tested that now, and that works, and that will make things consistent?

Reply:
Same with `linalg`, then?  So,
```
table_titles = ['NAME', 'COMPILER', 'MPI', 'LINALG', 'FFT']
. . . 
    col_names = {
        'NAME': 'Name',
        'COMPILER': 'Compiler(s)',
        'LINALG': "Linear algebra",
    }
```
and all references in the code to the text contents of `table_titles` would have the upper version.

Reply:
+1 for consistency, so going all upper is fine by me